### PR TITLE
fixed warnings

### PR DESCRIPTION
**Pull request by wrey75**
Fixed some warnings discovered during compilation with QtCreator.
Unused variable warning removal based on http://stackoverflow.com/questions/1486904/how-do-i-best-silence-a-warning-about-unused-variables

### CODE REVIEW

**Comment by ddennedy** [comment]
I just now noticed your updates as I did not receive email notification.
I was fine with not using Q_UNUSED for the mvcp code as that code is forked and the change could be applied to the original, non-Qt branch. However, for the remainder of the Qt-based code, we have already been using Q_UNUSED (but not thoroughly as you noticed). I would like to be consistent. At this time, I prefer non-mvcp code use the Q_UNUSED macro. Is there a reason you chose not to use it?

**Comment by wrey75** [comment]
I was not aware about Q_UNUSED macro provided by Qt. That's why I used the "void" form.
I will continue to merge the changes and analyse your code. I am really excited to have an open-source video editor.

**Comment by ddennedy** [comment]
Why did you close the pull request? Maybe you misunderstood my last comment. It was only regarding the change to src/src.pro, not all of the changes. I will merge your commits, squash them, and amend the result while still giving you credit.

**Comment by wrey75** [comment]
Basically, I have made a mistake: now the pull request contains much more stuff than expected and especially a big issue related to the path for the MLT framework. Because I have not isolated my changes in a specific branch I suppose. I am a newbie in both pull requests and Qt framework.
That's why I've closed the pull request.

File: src/src.pro [hunk 1]
```diff
@@ -524,7 +524,7 @@ mac {
     # QMake from Qt 5.1.0 on OSX is messing with the environment in which it runs
     # pkg-config such that the PKG_CONFIG_PATH env var is not set.
     isEmpty(MLT_PREFIX) {
-        MLT_PREFIX = /opt/local
```
Comment (wrey75) [inline src/src.pro]: Big mistake due to compilation issue I've encountred. I've committed the master branch too quickly.

**Pull request closed without merging**